### PR TITLE
Add the blog post for Tenzir v4.12

### PR DESCRIPTION
**Pull request by jachris**
This adds the blog post for an upcoming release.

### CODE REVIEW

File: web/blog/tenzir-v4.12/download-button.png
Comment (dominiklohmann) [inline web/blog/tenzir-v4.12/download-button.png:1]: Here's a compressed version of this file that's just 30% of the original size. We should always do that for PNGs. Please _force-push_ this to remove the old image from the commit history.

download-button-2

File: web/blog/tenzir-v4.12/index.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+{"src": "192.0.2.1", "num": 2}
+// after 1min
+{"src": "192.0.2.1", "num": 1}
+```
+
+In summary, these transformations provide powerful in-band capabilities leading
+to substantial data reduction, simplifying the analysis, and making data shaping
+more efficient.
+
+## Publish and subscribe
```
Comment (dominiklohmann) [inline web/blog/tenzir-v4.12/index.md:99]:
```suggestion
## Publish and Subscribe
```

File: web/blog/tenzir-v4.12/index.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+## Download Button
+
+Ever wanted to just save the output from Explorer to your computer? With the new
+**Download** button, you can do precisely that:
+
+![Download Button](download-button.png)
+
+Just select an available form and you're good to go!
+
+## Other Changes
```
Comment (dominiklohmann) [inline web/blog/tenzir-v4.12/index.md:159]: Here or above we should prominently mention the potentially breaking change to `context update` and others.

File: web/src/css/custom.scss [hunk 3]
```diff
@@ -105,6 +105,11 @@ img[alt$="#width600"] {
   fill: none;
 }
 
+/* We do not want font ligatures in code blocks. */
+code, pre {
+  font-variant-ligatures: none;
+}
+
```
Comment (jachris) [inline web/src/css/custom.scss:112]: I think this change is uncontroversial, but I want to point it out here so that there are no surprises.

**Review by dominiklohmann — APPROVED** [review]
Just nitpicks, ready to go other than that.

File: web/blog/tenzir-v4.12/index.md [hunk 4]
```diff
@@ -0,0 +1,180 @@
+---
+title: Tenzir v4.12
+authors: [mavam, jachris]
```
Comment (mavam) [inline web/blog/tenzir-v4.12/index.md:3]:
```suggestion
authors: [jachris]
```

File: web/blog/tenzir-v4.12/index.md [hunk 5]
```diff
@@ -0,0 +1,180 @@
...
+date: 2024-04-18
+tags: [tcp, udp, publish, subscribe, deduplicate, unroll, syslog, every]
+comments: true
+---
+
+We are thrilled to announce Tenzir
+[v4.12](https://github.com/tenzir/tenzir/releases/tag/v4.12.0), a feature-packed
+release introducing numerous enhancements. Notable additions include list
+unrolling, event deduplication, and the deployment of advanced pipeline
+architectures with publish-subscribe. We've also incorporated a download button,
```
[the remark this answers is not in the document]
Comment (mavam) [reply web/blog/tenzir-v4.12/index.md:13]: Fancy word for "add".

**Pull request merged**